detectable/src/main/java/com/synopsys/integration/detectable/detectables/gradle/inspection/parse/GradleReportLineParser.java

```diff
@@ -27,9 +25,10 @@ public class GradleReportLineParser {
     private static final String REQUIRE = "require";
     private static final String PREFER = "prefer";
     private static final String REJECT = "reject";
-    private static final Map<String, String> gradleRichVersions = new HashMap<>();
+    private static final Map<String, Map<String, String>> gradleRichVersions = new HashMap<>();
     private static final Map<String, HashSet<String>> richVersionGroup = new HashMap<>();
     private static final Map<String, String> relationsMap = new HashMap<>();
+    private static String foundParentProject = "";
```

Comment:
I missed this in previous reviews, but I find it a bit strange that some of these variables are static.
Might this be an issue if the class is used multiple times in a Detect execution?

Reply:
Good point, yeah I missed that too, will clean it up.